helm/examples/azure/README.md

```diff
@@ -1,5 +1,7 @@
 # HPCC Azure storage
 
+For Kubernetes server version 1.21+ the Azure Files Container Storage Interface (CSI) driver is used by Azure Kubernetes Service (AKS) to manage the lifecycle of Azure File shares. Currently most AKS regions are using Kubernetes version 1.21 and above. In the examples, Azure only no CSI driver storage will "Dynamically generate a storage account and shared volume with Storage Class" which will have a CSI provisioner by default. The user can select to use the legacy "kubernetes.io/azure-file" in the values.yaml.
```

Comment:
I think better, but "Azure only no CSI driver storage will" is not clear.
Does this mean - In Azure, if the csi.driver is left unset, it will .. ?

Reply:
On AKS Kubernetes 1.21+ CSI driver will be automatically installed by AKS and CSI class storage will be the default. For most Azure regions AKS will have Kubernetes 1.21+ as default. The stable Kubernetes version is 1.24 I think.